modules/ref_fb_module/src/statistics_fb_impl.cpp

```diff
@@ -503,10 +601,34 @@ void StatisticsFbImpl::calculate(
 
 void StatisticsFbImpl::onPacketReceived(const InputPortPtr& port)
 {
-    processPackets(port);
+    if (port.getLocalId() == "trigger")
```

Comment:
You can do object comparison here instead of String comparsion - if you compare 2 ObjectPtr to each other, unless they have `equals()` overridden, they'll check whether they point to the same object. This can be replaced with: 

```
if (port == triggerOutput)
    ...
```